### PR TITLE
updating readme files of priority components

### LINKED ISSUE #21: Review and improve element READMEs

### PR DESCRIPTION
**Pull request by kylebuch8**
Fixes #21

### CODE REVIEW

File: elements/rh-datetime/README.md [hunk 1]
```diff
@@ -1,6 +1,107 @@
...
+
+### minute
+
+Possible values: `numeric`, `2-digit`
+
+### second
+
+Possible values: `numeric`, `2-digit`
+
+Options are:
```
Comment (mwcz) [inline elements/rh-datetime/README.md]: @kylebuch8

Comment (mwcz) [reply elements/rh-datetime/README.md]: wat

File: elements/rh-number/README.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+
+The type of display you want to show.
+
+The options for type are:
+- `ordinal`: (1st, 2nd, 3rd, 4th)
+- `bytes`: (2 KiB, 9.54 Mib, 93 Gib)
+- `abbrev`: (1k, 1m, 1b)
+- `percent`: (10%, 50%, 100%)
+- `e`: (2.000e+6)
+- `thousands`: (321.123)
```
Comment (mwcz) [inline elements/rh-number/README.md]: should be a thin space (`&thinsp;` I think?), not `.`

File: elements/rhelement/README.md [hunk 3]
```diff
@@ -8,24 +8,27 @@ Here is an example of how RHElement should be extended
 ```
```
Comment (mwcz) [inline elements/rhelement/README.md]: mention generator and `npm run new` here, to be clear that's the correct way to create an element

**Comment by kylebuch8** [comment]
@mwcz - I think we're good here now. Can you review?

File: elements/rh-datetime/README.md [hunk 4]
```diff
@@ -1,6 +1,103 @@
...
+  month="long"
+  year="numeric">
+  Mon Jan 2 15:04:05 EST 2006
+</rh-datetime>
+```
+
+### With time: Monday, Jan 02, 2006, 3:04:05 PM
+```
+<rh-datetime
+  datetime="Mon Jan 2 15:04:05 EST 2006"
```
Comment (mwcz) [inline elements/rh-datetime/README.md]: I think there should be a time here.  `, 15:05:05` perhaps?

File: elements/rh-datetime/README.md [hunk 5]
```diff
@@ -1,6 +1,103 @@
...
+### datetime (observed)
+
+The value of this should be the same timestamp that you add to the light DOM.
+
+### type (observed)
+
+The options for type are:
+- `local`: Shows a formatted time for the indicated locale if provided
+- `relative`: Shows a relative time (1 hour ago, 2 hours until)
+
```
Comment (mwcz) [inline elements/rh-datetime/README.md]: Is there a default type?

File: elements/rh-icon/README.md [hunk 6]
```diff
@@ -1,5 +1,53 @@
...
+<rh-icon icon="rh-icon-server"></rh-icon>
+```
+
+## Attributes
+
+### icon (observed)
+
+The name of the icon that you want to use. If the value of this attribute changes, the new icon will show up in the UI.
+
+### data-size
```
Comment (mwcz) [inline elements/rh-icon/README.md]: I thought we were going to omit `data-` prefixes for custom attributes.

File: elements/rh-icon/README.md [hunk 7]
```diff
@@ -1,5 +1,53 @@
...
+
+### data-block
+
+By default, `rh-icon` is set to `display: inline-block`. The `data-block` attribute sets this element to `display: block`.
+
+```
+<rh-icon icon="rh-icon-server" data-block></rh-icon>
+```
+
+### data-bg
```
Comment (mwcz) [inline elements/rh-icon/README.md]: I'm going off the rails of this PR a bit, but I find it a little confusing that an attribute named `bg` can control both the color of the background (transparent or not) and also the shape (circle or square).

File: elements/rh-icon/README.md (hunk 6 above)
Comment (kylebuch8) [reply elements/rh-icon/README.md]: We'll fix this when we update the element.

File: elements/rh-datetime/README.md (hunk 5 above)
Comment (kylebuch8) [reply elements/rh-datetime/README.md]: No.

**Pull request merged**